File: trunk/proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/format/DuplicatedTemplateException.java
package com.jasonzqshen.familyaccounting.core.exception.format;

public class DuplicatedTemplateException extends Exception {

    /**
     * 
     */
    private static final long serialVersionUID = -7827938554803462945L;

    public DuplicatedTemplateException(int id) {
        super("Duplicated template identity " + id);
    }

}

File: trunk/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/DuplicatedTemplateException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class DuplicatedTemplateException : Exception
    {
        public DuplicatedTemplateException(int id)
            : base("Duplicated template identity " + id)
        {
        }
    }
}
